.allowed-scripts.mjs

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { configureAllowedScripts } from '@ministryofjustice/hmpps-npm-script-allowlist'
+
+export default configureAllowedScripts({
+  allowlist: {
+    'node_modules/@parcel/watcher@2.5.1': 'FORBID',
+    'node_modules/cypress@15.7.1': 'ALLOW',
```

Comment:
we're planning on going to playwright for our int test approach so this line will prob not be needed. But we'll refactor once we do that implementation